### DISCUSSION TITLE
New Image Viewer

### DISCUSSION THREAD

**Discussion by ostap667inbox** [civitai/civitai#210] [Ideas]
In the new image viewer, the interface element showing how many and which images are open has disappeared. Instead, there is not even a primitive "Image 3 of 20".
Users will simply scroll through the pictures in a circle, not knowing where the beginning and where the end is. This is very uncomfortable. Especially if you save the pictures to yourself as examples for the downloaded model.

**Comment by JustMaier** [comment]
Thanks for bringing this to our attention, we noticed this too and it's been fixed 👍
